src/main/java/com/lpvs/controller/LPVSWebController.java

```diff
@@ -60,4 +59,19 @@ public LPVSLoginMember loginMember(Authentication authentication) {
         }
     }
 
+    @CrossOrigin(origins = "http://localhost:3000")
```

Comment:
Please consider to set this in properties.

Reply:
Done.